### PR TITLE
HCLIND-4 Fix: Hardcoded TLD in cloud_cost_anomaly_alerts.pt to US

### PR DESCRIPTION
**Pull request by juanestrahcl**
### Description

Get dynamic TLD depending of current "rs_governance_host".

### Issues Resolved

Link to the report seems to be hardcoded to US shard.

### Contribution Check List

- [ ] All tests pass.
- [ ] New functionality includes testing.
- [x] New functionality has been documented in the README if applicable
- [x] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

**Review by taniaSol — COMMENTED** [review]
- Maybe replace changelog entry with more readable message, such as "URL link to Cost Anomalies report is constructed dynamically based on detected shard"
- There is no need to updated readme with domain details as this is an out-of-the-box policy and from user point of view it should work in all Flexera shards
- The logic to construct the report URL won't work in staging

**Comment by juanestrahcl** [comment]
> * Maybe replace changelog entry with more readable message, such as "URL link to Cost Anomalies report is constructed dynamically based on detected shard"
> * There is no need to updated readme with domain details as this is an out-of-the-box policy and from user point of view it should work in all Flexera shards
> * The logic to construct the report URL won't work in staging

@taniaSol about last point, why it won't work in staging?

File: cost/cloud_cost_anomaly_alerts/cloud_cost_anomaly_alerts.pt [hunk 1]
```diff
@@ -161,7 +161,7 @@ datasource "ds_get_anomalies" do
 end
 
 datasource "ds_filter_anomalies" do
-  run_script $js_filter_anomalies, $ds_get_anomalies, $param_min_spend, rs_org_id, $param_days, $param_dimensions, $param_cost_metric, $param_anomaly_limit
+  run_script $js_filter_anomalies, $ds_get_anomalies, $param_min_spend, rs_org_id, $param_days, $param_dimensions, $param_cost_metric, $param_anomaly_limit, f1_app_host
```
[the remark this answers is not in the document]
Comment (juanestrahcl) [reply cost/cloud_cost_anomaly_alerts/cloud_cost_anomaly_alerts.pt:164]: It's a reserved word, it isn't listed on docs ([Reserved Word Reference](https://docs.flexera.com/flexera/EN/Automation/ReservedWordReference.htm)) so maybe we should add it, but it's listed on governance repo ([f1_app_host](https://github.com/rightscale/governance/blob/master/eval/keyword.go#L54)), I tested it and these are some values it returns:

`f1_app_host` of  _https://app.flexera.com_ = **app.flexera.com**
`f1_app_host` of  _https://app.flexeratest.com_ = **app.flexeratest.com**

[the remark this answers is not in the document]
Comment (nia-vf1) [reply cost/cloud_cost_anomaly_alerts/cloud_cost_anomaly_alerts.pt:164]: Does this also support EU tenants?

Maybe using `rs_governance_host` makes more sense - @bryankaraffa what are your thoughts? 

@taterdog590 fyi

[the remark this answers is not in the document]
Comment (juanestrahcl) [reply cost/cloud_cost_anomaly_alerts/cloud_cost_anomaly_alerts.pt:164]: Previously I was using `rs_governance_host`, @Haydenmarx and @taniaSol told us we should use `f1_app_host` instead.

[the remark this answers is not in the document]
Comment (nia-vf1) [reply cost/cloud_cost_anomaly_alerts/cloud_cost_anomaly_alerts.pt:164]: Understood, thanks. 

@taniaSol @Haydenmarx does `f1_app_host` support EU endpoints as well?

[the remark this answers is not in the document]
Comment (nia-vf1) [reply cost/cloud_cost_anomaly_alerts/cloud_cost_anomaly_alerts.pt:164]: Got confirmation 👍

**Review by nia-vf1 — APPROVED** [review]
LGTM

**Comment by juanestrahcl** [comment]
Code was tested on North America (app.flexera.com & app.flexeratest.com), EU (app.flexera.eu) and APAC (app.flexera.au), in all regions it's working well.

**Pull request merged**